server/client.go

```diff
@@ -1536,20 +1606,33 @@ func (c *client) queueOutbound(data []byte) bool {
 }
 
 // Assume the lock is held upon entry.
-func (c *client) sendProto(info []byte, doFlush bool) {
-	if c.nc == nil {
+func (c *client) enqueueProtoAndFlush(proto []byte, doFlush bool) {
```

Comment:
I get the change but.. ;)

I think sendProto is ok here with comments about what it does.

Reply:
But since you have higher level functions that look a bit nicer probably ok.

Reply:
The goal really was to avoid issues that if you made sendProto() calls in the JS branch, I want you to have to resolve the conflict and understand that the new call does now. Wanted also to make sure that the memory muscle of using sendProto() in the future fails :-)

Reply:
Good point, and agree now that I reviewed it all.